update.sh

```diff
@@ -2,22 +2,86 @@
 set -e
 set -x
 
-# download and unpack latest release
-mvn dependency:unpack \
--DremoteRepositories=https://repo.adeo.no/repository/maven-releases/ \
--Dartifact=no.nav.abac.policies:abac-attribute-constants:RELEASE:jar:sources \
-
-# replace src-directory
-rm -rf src/
-mkdir -p src/main/java
-mv target/dependency/no src/main/java
+parent_version=`mvn help:evaluate -Dexpression=project.version -q -DforceStdout`
+
+modules=(
+    "abac-common-attributes-alfa"
+    "abac-veilarb-policies-alfa"
+)
+
+##############################################################
+# TODO remove when release points to a true release candidate
```

Comment:
@atlemauland could you please release these packages so that the current release candidate points to a version with an attributes-artifact?